specification/web/resource-manager/Microsoft.Web/stable/2020-10-01/Provider.json

```diff
@@ -212,6 +422,270 @@
+          "type": "boolean"
+        },
+        "appInsightsSettings": {
+          "$ref": "#/definitions/AppInsightsWebAppStackSettings",
+          "description": "Application Insights settings associated with the minor version."
+        },
+        "gitHubActionSettings": {
+          "$ref": "#/definitions/GitHubActionWebAppStackSettings",
+          "description": "GitHub Actions settings associated with the minor version."
+        },
+        "appSettingsDictionary": {
+          "description": "Application settings associated with the minor version.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "siteConfigPropertiesDictionary": {
+          "$ref": "#/definitions/SiteConfigPropertiesDictionary",
+          "description": "Configuration settings associated with the minor version."
+        },
+        "supportedFunctionsExtensionVersions": {
+          "description": "List of supported Functions extension versions.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "isPreview": {
+          "description": "<code>true</code> if the stack is in preview; otherwise, <code>false</code>.",
+          "type": "boolean"
+        },
+        "isDeprecated": {
+          "description": "<code>true</code> if the stack is deprecated; otherwise, <code>false</code>.",
+          "type": "boolean"
+        },
+        "isHidden": {
+          "description": "<code>true</code> if the stack should be hidden; otherwise, <code>false</code>.",
+          "type": "boolean"
+        },
+        "endOfLifeDate": {
+          "description": "End-of-life date for the minor version.",
+          "type": "string"
+        },
+        "isAutoUpdate": {
+          "description": "<code>true</code> if the stack version is auto-updated; otherwise, <code>false</code>.",
+          "type": "boolean"
+        },
+        "isEarlyAccess": {
+          "description": "<code>true</code> if the minor version is early-access; otherwise, <code>false</code>.",
+          "type": "boolean"
+        },
+        "isDefault": {
+          "description": "<code>true</code> if the minor version the default; otherwise, <code>false</code>.",
+          "type": "boolean"
+        }
+      }
+    },
+    "FunctionAppRuntimes": {
+      "description": "Function App stack runtimes.",
+      "type": "object",
+      "properties": {
+        "linuxRuntimeSettings": {
+          "$ref": "#/definitions/FunctionAppRuntimeSettings",
+          "description": "Linux-specific settings associated with the minor version."
+        },
+        "windowsRuntimeSettings": {
+          "$ref": "#/definitions/FunctionAppRuntimeSettings",
+          "description": "Windows-specific settings associated with the minor version."
+        }
+      }
+    },
+    "FunctionAppStack": {
```

Comment:
should we add some `readonly` to this schema?

Same for other schema that seems only used in return value.